infrastructure/template.yaml

```diff
@@ -48,24 +45,11 @@ Parameters:
     Default: "third-party-stubs"
 
 Conditions:
-  EnforceCodeSigning: !Equals
-    - !Ref CodeSigningEnabled
-    - true
-  UseCodeSigningConfigArn:
-    Fn::Not:
-      - Fn::Equals:
-          - !Ref CodeSigningConfigArn
-          - "none"
```

Comment:
Why has the code signing config been changed?

Reply:
cfn-lint issues, you had 2 codesigning config lookups 1 of which wasnt used, so I changed it to the "official" way to do it

Reply:
Maybe it can be `Default: ""` see https://github.com/aws/aws-sam-cli/pull/1785